Framework/Algorithms/inc/MantidAlgorithms/CorelliPowderCalibrationApply.h

```diff
@@ -0,0 +1,68 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2014 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+
+#pragma once
+
+#include "MantidAPI/Algorithm.h"
+#include "MantidAPI/ITableWorkspace.h"
+#include "MantidAPI/MatrixWorkspace.h"
+#include "MantidAlgorithms/CorelliPowderCalibrationDatabase.h"
+#include "MantidAlgorithms/DllConfig.h"
+#include "MantidDataObjects/EventWorkspace.h"
+#include "MantidDataObjects/TableWorkspace.h"
+
+namespace Mantid {
+namespace Algorithms {
+/**
+ * @brief Apply calibration table for Corelli Powder Diffraction
+ *
+ */
+class MANTID_ALGORITHMS_DLL CorelliPowderCalibrationApply
+    : public API::Algorithm {
+public:
+  /// Algorithm's name for identification
+  const std::string name() const override {
+    return "CorelliPowderCalibrationApply";
+  };
+
+  /// Summary of algorithm's purpose
+  const std::string summary() const override {
+    return "Apply Corelli calibration results onto input workspace";
+  };
+
+  /// Algorithm's version, overriding a virtual method
+  int version() const override { return 1; };
+
+  /// Algorithm's category, overriding a virtual method
+  const std::string category() const override {
+    return "Diffraction\\Calibration";
+  };
+
+  /// Extra help info
+  const std::vector<std::string> seeAlso() const override {
+    return {
+        "CorelliPowderCalibrationGenerate & CorelliPowderCalibrationDatabase"};
```

Comment:
Can you include also  `CorelliPowderCalibrationLoad`?

Reply:
yep, it will show up in the next batch of commits.